File: internal/domain/models/video.go
package models

import "gorm.io/gorm"

type Video struct {
	gorm.Model
	Name    string `json:"Name"`
	Path    string `json:"Path"`
	TitleId uint   `json:"TitleId"`
	Episode uint   `json:"Episode"`
	Season  uint   `json:"Season"`
}

File: gui/src/types/Video.ts
export interface Video {
    ID:        number;
    CreatedAt: Date;
    UpdatedAt: Date;
    Name:      string;
    Path:      string;
    IsDir:     boolean;
    TitleId:   number;
    Episode:   number;
    Season:    number;
    Ext:       string;
}